md/howto-debugcustompage.md

```diff
@@ -0,0 +1,89 @@
+# How to Debug a custom page ? 
+
+<h2></h2>
+
+##  Question
+
+The engine load the custom page, and then you want to debug it : change an item in the Look And Feel, modify the Groovy code or whatever. The engine has a mode that you can do that directly, and just click on the portal to reload the page and take into account the change without zip and deploy your change.
+   
+
+## How
+::: info
+This procedure is the same for a Server, or for the Bonita Engine behind the Bonita Studio
+:::
+
+### Turn the debug mode in the engine.
+This procedure is valid for a server, or for the server behind the studio.
+
+First, the engine must be turn in the "debug mode". In this mode, the engine reload the page each time you access it (for a custom page, when you click on the menu bar, for a RestAPIExtension, when you access it).
+
+::: info
+If you do that on a fresh installation, before you start the server for the first time, it's better to follow the Step "Initial setup". If the server was already started (which is the situation when you use the Bonita Studio) follow the procedure "Change the setup"
+:::
+
+#### Initial setup
+The change must be done before the first startup.
```

Comment:
this implies you edit the "template" portal file, since your tenant file does not exist at this time